src/main/scala/is/hail/io/vcf/ExportVCF.scala

```diff
@@ -75,9 +92,19 @@ object ExportVCF {
     case TChar => "Character"
     case TString => "String"
     case TBoolean => "Flag"
+    case _ => fatal(s"Cannot export type `$t' to VCF INFO field.")
+  }
 
-    // FIXME
-    case _ => "String"
+  def formatType(t: Type): String = t match {
+    case TArray(elementType) => formatType(elementType)
+    case TSet(elementType) => formatType(elementType)
+    case TInt => "Integer"
+    case TDouble => "Float"
+    case TChar => "Character"
+    case TString => "String"
+    case TBoolean => "String"
```

Comment:
I wouldn't implicitly convert Boolean to String.  Make the user do it if they want a non-supported Boolean in the FORMAT field.